packages-ts/gauntlet-terra-contracts/src/commands/contracts/proxy_ocr2/confirmContract.ts

```diff
@@ -13,8 +14,12 @@ type ContractInput = {
 }
 
 const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  const rdd = getRDD(flags.rdd)
+  const contract = args[0]
+  const aggregator = rdd.contracts[contract]
```

Comment:
We don't need RDD on these commands, just the contract address.

Reply:
But we still want to verify that the contract address exists in the RDD right?

Reply:
nvm there was a bug of passing in the whole template into `address` which gives validation error